app/models/order_cycle.rb

```diff
@@ -3,6 +3,8 @@
 require 'open_food_network/scope_variant_to_hub'
 
 class OrderCycle < ApplicationRecord
+  acts_as_taggable
```

Comment:
Why don't we re-use the existing tagging functionality? Thoughts:

* Tagging order cycles is a simper data structure and it's easier to work with here.
* But we need to add more code to deal with order cycle tags.
* Simple order cycles still have exchanges we can tag.
* If we convert a simple order cycle to a complex one then the simple tags are suddenly hidden and can't be edited?